src/handbook/development/releases/process.md

```diff
@@ -3,85 +3,161 @@ navTitle: Release Process
 ---
 # Release Process
 
-There are two processes, one for major/minor point releases. The second for patch
-releases.
+We do two types or releases:
 
-## Major/minor point releases
+ - Scheduled releases at a regular [monthly cadence](./planning.md#cadence). This
+   encompasses the entire set of product repositories.
+ - Fix releases of individual repositories as needed
```

Comment:
```suggestion
 - [Scheduled Releases](#scheduled-releases) - _for the planned release of the entire set of product repositories._
 - [Fix Releases](#fix-releases) - _for ad-hoc release of individual repositories._
```

(Suggestion)

1. Link to the sections mentioned 
2. Remove the link to planned cadence as it is linked immediately below in the actual section